template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/yarn/YarnVolumeConfigProviderTest.java

```diff
@@ -70,7 +70,7 @@ void getRoleConfigsWithMultipleEphemeralVolumes() {
     @Test
     void getRoleConfigsWithMultipleAttachedVolumes() {
         HostgroupView worker = hostGroupWithVolumeTemplatesAndTemporaryStorage(2,
-                Sets.newHashSet(new VolumeTemplate()), TemporaryStorage.ATTACHED_VOLUMES, 0);
+                Sets.newHashSet(new VolumeTemplate()), TemporaryStorage.ATTACHED_VOLUMES, 0, 00);
```

Comment:
double 0

Reply:
Yes, I will change it to single 0. Made a mistake with it.